src/Routing/RouteBuilder.php

```diff
@@ -149,14 +149,24 @@ public function routeClass($routeClass = null)
      * extensions applied. However, setting extensions does not modify existing routes.
      *
      * @param null|string|array $extensions Either the extensions to use or null.
+     * @param boolean $merge If TRUE `$extensions` will be merged with current
      * @return array|null
      */
-    public function extensions($extensions = null)
+    public function extensions($extensions = null, $merge = true)
```

Comment:
Taking in consideration that accessing the collection is an extremely obscure thing to do